packages/jest-serializer-make-styles/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import { DEFINITION_LOOKUP_TABLE, RULE_CLASSNAME_INDEX } from '@fluentui/make-styles';
```

Comment:
nit: Might be worth linking to some docs for future maintenance by someone who might not know what they are doing (i.e me): https://github.com/facebook/jest/blob/master/packages/pretty-format/README.md#serialize

Reply:
or in the README even :)